collections/_components/text-input.html

```diff
@@ -2,6 +2,249 @@
 title: Text Input
 description: Text inputs allow your users to enter letters, numbers and symbols on a single line.
 layout: layout-sidenav
-category: form
-react: true
----
\ No newline at end of file
+category: general
+react-link: true
+vue-link: true
+github-link: true
+toc: true
+---
+
+{% capture with-hint-icon %}
+<div class="sgds row">
+    <div class="col form-group has-feedback">
+        <p>Default</p>
+        <label for="defaultWith" class="form-label">Label</label>
```

Comment:
Can remove this
```<p>Default</p>```

and use the label to display the different state. Same for the rest of the examples
```<label for="defaultWith" class="form-label">Default Input</label>```